lib/api/src/grpc/proto/points.proto

```diff
@@ -402,8 +402,8 @@ message GeoBoundingBox {
 }
 
 message GeoRadius {
-  GeoPoint center = 1; // Center of the circle
```

Comment:
those two changes are British English, I'd rather stick to the American variants please